### DISCUSSION TITLE
Editing tables within markdown and HTML?

### DISCUSSION THREAD

**Discussion by reagle** [saulpw/visidata#1503] [Q&A] [answered]
I read that [vd supports markdown](https://www.visidata.org/docs/formats/), but it is not able to detect any of the tables in <https://reagle.org/joseph/2016/min-eg/test-tables.md>. When I convert that to HTML, <https://reagle.org/joseph/2016/min-eg/test-tables.html>, it can load the tables but not edit them. I know I could save copy, edit, and save them elsewhere, but wouldn't that drop any other HTML?

1. What kind of markdown (and tables) can vd read?
2. Can vd edit embedded tables *within* their documents?

**Comment by anjakefala** [comment, marked as answer]
1. VisiData can save markdown tables! It cannot yet load them; no one has championed that feature. The save as markdown feature was one I requested for maintaining tables like https://visidata.org/docs/formats. =)

VisiData uses `lxml` to parse html tables with a `<table>` tag. There are known gaps of support: https://github.com/saulpw/visidata/issues/1308 , https://github.com/saulpw/visidata/issues/1346.

**Reply by reagle** [reply]
Okay, I can see that being able to edit tables within markdown (while retaining non-table content on save) is not possible.
